src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader/ClusterDetailsEventProcessor.java

```diff
@@ -122,17 +122,19 @@ public static NodeDetails getCurrentNodeDetails() {
   }
 
   public static class NodeDetails {
-
     private String id;
     private String hostAddress;
     private String role;
+    private Boolean isMasterNode;
```

Comment:
can we use boolean ?

Reply:
This value can even be null so we will need Boolean